blockchain/tx_pool.go

```diff
@@ -1025,6 +1025,7 @@ func (pool *TxPool) validateBlobTx(tx *types.Transaction) error {
 		return errors.New("missing sidecar in blob transaction")
 	}
 	// Ensure the blob fee cap satisfies the minimum blob gas price
+	blobTxMinBlobGasPrice := new(big.Int).Mul(pool.gasPrice, blobBaseFeeMultiplier)
```

Comment:
Can we use eip4844.go CalcBlobFee instead? We'll need to verify these in multiple places.